### ISSUE TITLE
About updating data

### ISSUE THREAD

**Issue by rafaelsabino** [cipriancraciun/covid19-datasets#34]
Good Morning. My question is about thanks and not any other type of error. For a long time I scanned the internet for a reliable source and a database form to use in my web application and now I finally met you. I am very happy to have found this repository. I would like to know if there is a possibility that someday you will not be updated or if you do not want this work anymore, because I am using your data grouped for my analysis and it would be very good to be able to count on this incredible update from you. You have my admiration. This work you did is fantastic and here in Brazil it will help us a lot in the analysis of covid19. When I finish this job how do I reference your repository? Ah you can close this issue when you read. A big hug.

**Comment by cipriancraciun** [comment]
:)  Thanks for the kind words.  I am glad I could have helped others.

----

Regarding the updating of the datasets:
* all the code that does the processing and plotting is automated;  (I.e. it requires a single tool invocation;)
* the actual execution could be done automatically, however due to continuous inconsistencies for the JHU datasets, I manually run this each morning (at around 05 UTC), and check the logs for any issues, and do a quick diff of the `tsv` files to see if something is very wrong;
* I also run this procedure from time to time during the day, as the ECDC and NY Times have other release schedules;

Therefore, I foresee running these updates daily at least for the immediate future (say one or two months) until the whole thing cools down.  Afterwards, I'll run the updates perhaps a couple of times a week.

----

Regarding the data files, at the moment many of the files are uploaded uncompressed.  However GitHub has a hard-limit on 100 MiB for files, therefore sometimes perhaps next week I'll be forced to switch to `gzip` compression for selected files.

If you'll get an error about fetching the files, check if perhaps this isn't the cause.

I see that you have already registered as "watching" this repository, thus I'll post an issue here when this change would happend.

----

Regarding on how to cite this work, I've created a section in the readme:
* https://github.com/cipriancraciun/covid19-datasets#attribution
* basically it says that you should first cite the actual data source (i.e. JHU, ECDC, NY Times, etc.), and then if you want and have space you could point to this repository;  for example:
~~~~
based on original data from JHU CSSE (https://github.com/CSSEGISandData/COVID-19),
as processed and augmented at https://github.com/cipriancraciun/covid19-datasets
~~~~

BTW, when you finish your work, let me know and I'll point to it in the `readme.md`.

----

If you have any questions or issues don't hesitate to open a ticket on this repository.

**Comment by rafaelsabino** [comment]
Good afternoon. thank you very much for the answers. May God bless your work. This will help thousands of people. When finished, yes, I'll let you know. I will close this issue so as not to disturb your activities. I am very grateful to you. :)
